### ISSUE TITLE
Some sightings are showing "invalid date"

### ISSUE THREAD

**Issue by nickdos** [AtlasOfLivingAustralia/pigeonhole#37]
Trying to edit these sightings results in an error page with stacktrace:

```
2015-11-20 05:27:35,087 ERROR [GrailsExceptionResolver] IllegalArgumentException occurred when processing request: [GET] /edit/e1d6d7b8-10f6-46b7-9d2f-50d17811f48d
Cannot format given Object as a Date. Stacktrace follows:
org.codehaus.groovy.grails.web.pages.exceptions.GroovyPagesException: Error processing GroovyPageView: Cannot format given Object as a Date
```

I suspect the date is coming from EXIF data.

**Comment by nickdos** [comment]
Related to this issue is report from user:

> Attached file is one in question, the problem only occurred when I changed time-zones on my Mac, not sure how to replicate it now, I changed to central australian time when I went over into SA.
> 
> Below is the couple of lines that got me thinking it was the TZ,
> 
> xmp:CreateDate2015-11-10T18:49:20/xmp:CreateDate
> xmp:MetadataDate2015-11-16T20:51:51+10:30/xmp:MetadataDate
> 
> The second line had +.5:30 when I changed back to NSW time I got the above result and photo’s uploaded properly in sightings.
> 
> Now the other issue…when you mentioned my 2 sightings, I remembered they weren’t added as sightings, (those were just a couple I was mucking around with) they were done as a Data Set under Steve Dew’s Aussie Creatures data set or something like that…and there were a couple of hundred of them.
> 
> Steve

![chestnut rumped thornbill_d3_0988](https://cloud.githubusercontent.com/assets/532845/11460651/8c06f8ca-9744-11e5-9f49-21dd32006385.jpeg)

**Comment by nickdos** [comment]
Partial fix in 5a5ca1a
